src/event.rs

```diff
@@ -160,6 +160,12 @@ pub enum WaEvent {
         /// The URL to upload the file to.
         url: String
     },
+    MediaConn {
+        uuid: Uuid,
+        auth: String,
+        ttl: NaiveDateTime,
```

Comment:
Why isn't this `ttl` field a `Duration` (seeing as it seems to come in as one, but then you add it to the current timestamp)? Maybe there's something I'm missing though...

Reply:
Time to live refers to when the authorization data stops being valid. If you represent that using a Duration, then that does not make sense on its own, because you need to know the "starting point" of the duration.

Sooner or later you have to add TTL to the "starting point". On top of that, the user might not fetch the event right away. Thats why I decided to convert the TTL into NaiveDateTime as soon as I get the response from the server. Ensuring we are safe from issues steaming from delays.

Reply:
Okay, makes sense; thanks!